  소스.cpp
소스.obj : error LNK2019: _lua_close_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _lua_settop_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _lua_tolstring_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _lua_pcallk_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _luaL_loadbufferx_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _luaL_newstate_main 함수에서 참조되는 확인할 수 없는 외부 기호
소스.obj : error LNK2019: _luaL_openlibs_main 함수에서 참조되는 확인할 수 없는 외부 기호
lua54.lib : warning LNK4272: 'x64' 라이브러리 컴퓨터 종류가 'x86' 대상 컴퓨터 종류와 충돌합니다.
C:\Users\undug\Desktop\lua-test\Debug\lua-test.exe : fatal error LNK1120: 7개의 확인할 수 없는 외부 참조입니다.
